chat/views.py

```diff
@@ -15,3 +15,7 @@ class ChatView(generics.ListAPIView):
     pagination_class = LimitOffsetPagination
     lookup_field = 'channel_id'
     queryset = Chat.objects.order_by('-id').all()
+
+    def get_queryset(self):
+        cid = self.kwargs['channel_id']
+        return Chat.objects.order_by('-id').filter(channel_id=cid)
```

Comment:
i wonder there might be better way for cashing,
like, you already create queryset, so just,
`return queryset.filter(channel_id=cid)`
will be better or not

Reply:
pr범위 아니라서 나중에 고칠게

Reply:
1. 정확하게 말하자면, 여기서 쓰이는 `get_queryset` 메서드는 drf에서 정의된 메서드임으로, 장고 공식 document에서 예시를 가져오는 것은 말이 맞지 않음.
2. https://www.django-rest-framework.org/api-guide/generic-views/#get_querysetself 링크에 따르면 "This method should always be used rather than accessing self.queryset directly, as self.queryset gets evaluated only once, and those results are cached for all subsequent requests." 라고 나와있는데, `self.queryset`을 직접적으로 사용하는 것 보다 `self.get_queryset()`을 먼저 사용하라고 나와있음.

이를 바탕으로 `self.get_queryset()`으로 수정 예정.

@ds1sqe

Reply:
@ds1sqe 
아 내가 이해를 잘못 했네. 일단 더 알아보고 지적한대로 고쳐볼게~

Reply:
수정 완료 https://github.com/Team-Discipline/Xlack-Backend/pull/45/commits/2a5478df022cebfd0d0a628ef9ee376acb9bbfb8
@ds1sqe